Resources/Prototypes/Entities/Structures/Doors/Airlocks/external.yml

```
@@ -20,6 +20,9 @@
     layoutId: AirlockExternal
   - type: Paintable
     group: null
+  - type: Tag # imp
+    tags: 
+    - ExternalAirlock
```

needs to include parent tags
```suggestion
  - type: Tag # imp
    tags: 
    - Structure
    - Airlock
    - ExternalAirlock
```